CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing to Xournal++
+
+Thanks for your interest in contributing to Xournal++! The project is run
+completely by volunteers in their spare time, so any contribution--no matter how
+small--is greatly appreciated. In this file, we'll outline how you can
+contribute to the project.
+
+Xournal++ development primarily occurs on [GitHub at the xournalpp/xournalpp
+repository](https://github.com/xournalpp/xournalpp). As a contributor, you
+probably have a particular bug or feature that are you interested in working on.
+Before you do anything, you should first look in the [issue
+tracker](https://github.com/xournalpp/xournalpp/issues) to see if anyone has
+reported your bug or has a similar idea for a feature. If not, you should first
+submit a new issue detailing what you are about to do. This will allow you to
+get feedback and ensure that you do not end up duplicating work. You can obtain
+additional help by contacting community members through one of our [official
+communication channels](https://xournalpp.github.io/community/help/).
+
+## Contributing translations
+
+If you would like to contribute translations, you can submit improvements to
+[our project on Crowdin](https://crowdin.com/project/xournalpp).
```

Comment:
Maybe add a few lines on when we merge these translations.

Reply:
@LittleHuba like  that?
```suggestion
[our project on Crowdin](https://crowdin.com/project/xournalpp).
Those are then merged directly before the next release. 
So they wont appear directly after your contribution.
```